src/legacy/ui/public/vis/editors/default/components/default_editor_agg_select.tsx

```diff
@@ -80,42 +77,58 @@ function DefaultEditorAggSelect({
       <FormattedMessage
         id="common.ui.vis.defaultEditor.aggSelect.helpLinkLabel"
         defaultMessage="{aggTitle} help"
-        values={{ aggTitle: isAggTypeDefined ? value.title : '' }}
+        values={{ aggTitle: value ? value.title : '' }}
       />
     </EuiLink>
   );
 
+  const errors = [];
+
+  if (!aggTypeOptions.length) {
+    errors.push(
+      i18n.translate('common.ui.vis.defaultEditor.aggSelect.noCompatibleAggsDescription', {
+        defaultMessage:
+          'The index pattern {indexPatternTitle} does not contain any of aggregations.',
```

Comment:
```suggestion
          'The index pattern {indexPatternTitle} does not contain any aggregations.',
```

Reply:
@maryia-lapata There is still this typo ^^

Reply:
@cchaos I based on the latest [comment](https://github.com/elastic/kibana/pull/34548#discussion_r274067527)

> Same comment as above. The index pattern {indexPatternTitle} does not contain any of aggregations.

Should I get rid of `of`?

Reply:
Yes, I think she just didn't notice.

Reply:
Thank you for the comments. 
The message was updated.